queries.sql

```diff
@@ -1,3 +1,36 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+//Find all animals whose name ends in "mon".
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+
+//List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+
+
+//List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT * FROM animals WHERE neutered='True' AND escape_attempts<3;
+
+
+//List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT * FROM animals WHERE name='Agumon' OR name='Pikachu';
+
+
+//List name and escape attempts of animals that weigh more than 10.5kg
+SELECT * FROM animals WHERE weight_kg>10.5;
```

Comment:
- [ ] Here we have to select the name & escape_attempts.
`SELECT name, escape_attempts FROM animals WHERE weight_kg > '10.5';`

Reply:
Hello Deepakdanger,
The same thing with the above command ran, with 
![No output3](https://user-images.githubusercontent.com/101729024/205955416-c446db03-fc64-452b-89b9-c222ea731469.png)
no display of anything. Find attached. what should I do, pls.